.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5502
```

Comment:
We should have `.vscode` in our .gitignore file, but just as a heads up, this file shouldn't be checked in. No problem that it snuck through though.